x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_actions/actions/component/index_item.tsx

```diff
@@ -62,9 +63,9 @@ export const UsageItem: React.FC<UsageProps> = ({ usageItem }) => {
             </EuiFlexGroup>
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
-            <EuiLink data-test-subj="navigateToIndexPage" onClick={handleNavigateToIndex}>
+            <EuiLink data-test-subj="navigateToIndexPage" onClick={navigateToIndex}>
               <EuiIcon size="s" type="popout" />
-            </EuiLink>
+            </EuiLink>{' '}
```

Comment:
did you add this `{' '}` on purpose?

Reply:
Somehow this got added by `eslint`. It is removed now.